Assets/HoloToolkit-Examples/SpatialUnderstanding/SpatialUnderstanding-FeatureOverview/Scripts/AppState.cs

```diff
@@ -1,11 +1,14 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using UnityEngine;
 using HoloToolkit.Unity;
-using System;
 using HoloToolkit.Unity.InputModule;
 using HoloToolkit.Unity.SpatialMapping;
+using System;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
I think getting the guidance clarified would be good. 

Linq isn't the devil or anything, but I'm just encouraging the Microsoft recommendations.  That being said, if you feel like it's easier to complete the task with Linq without incurring a performance hit, then I don't see why you couldn't use it. I just wanted to get clarification on its use.